lib/themes/dosomething/paraneue_dosomething/js/components/SlideshowController.js

```diff
@@ -6,15 +6,18 @@ const React = require('react');
  */
 class SlideshowController extends React.Component {
   render() {
+    const disablePrevButton = this.props.currentSlide === 0 ? true : false;
+    const disableNextButton = this.props.currentSlide === (this.props.totalSlides - 1) ? true : false;
```

Comment:
Nitpick: the ternary doesn't do anything here (since it's just setting `true` if true, and `false` if false).